beesdoo_shift/models/planning.py

```diff
@@ -205,6 +207,7 @@ def _set_duration(self):
         if self.start_time:
             self.end_time = self.start_time + self.duration
 
+    @api.model
```

Comment:
Hum, not sure about that. I think it should be `@api.multi` as it is by default. Can you explain why, you chose `@api.model` ?